### DISCUSSION TITLE
Is the navigation folder missing?

### DISCUSSION THREAD

**Discussion by bllshttng** [chen-rn/CUA#54] [Q&A]
Hey @chen-rn awesome stuff. This looks dope. 

It seems there is no navigation folder in the app folder, nor in the app/provider folder. Am I missing something here? I'm happy to grab it from tamagui's sample project, but just wanted to make sure...

**Comment by albbus-stack** [comment]
Actually, since we switched to Expo Router, the navigation folder is not required anymore. We are going to update the readme asap :octocat:
